packages/bazel-bot/docker-image/generate-googleapis-gen.sh

```diff
@@ -103,7 +103,8 @@ for (( idx=${#ungenerated_shas[@]}-1 ; idx>=0 ; idx-- )) ; do
     # APIs from being updated.
     set +e
     # Invoke bazel build.
-    (cd "$GOOGLEAPIS" && bazelisk build $BAZEL_FLAGS -k $targets)
+    # --experimental_convenience_symlinks=normal is to make sure bazel-bin symlink is created
+    (cd "$GOOGLEAPIS" && bazelisk build --experimental_convenience_symlinks=normal $BAZEL_FLAGS -k $targets)
```

Comment:
The `experimental_` prefix on this flag name tells me not to depend on it for the long term.  I expect it to be dropped or renamed within a year or two.

Therefore, I'd prefer invoking `BAZEL_BIN=$(bazel info bazel-bin)` as you described in the issue.

Reply:
Done, PTAL.  Also, how do I test it properly?